package.json

```diff
@@ -76,9 +76,9 @@
     "babel-plugin-transform-object-rest-spread": "^6.23.0",
     "babel-preset-latest": "^6.22.0",
     "babel-preset-react": "^6.22.0",
-    "react": "^15.5.4",
-    "react-dom": "^15.5.4",
-    "react-scripts": "^0.8.4"
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "react-scripts": "^0.7.0"
```

Comment:
Keep all react dependencies consistent with peer dependencies until they we upgrade all react related dependencies all together to react 15.5: https://github.com/eggheadio/egghead-ui/issues/108